inc/dbmysql.class.php

```diff
@@ -762,6 +762,9 @@ function __construct ($dbconnexion, $table, $crit="", $debug=false) {
 
       $this->conn = $dbconnexion;
       if (is_string($table) && strpos($table, " ")) {
+//          if ($_SESSION['glpi_use_mode'] == Session::DEBUG_MODE) {
```

Comment:
Is ther any issue existing for this problem? I did not know that something has been implemented... It would be interesting I guess to add a word about error reporting in the dev docs as well, don't you?